assemblyscript/examples/signatures/README.md

```diff
@@ -72,19 +73,19 @@ The WASA wrapper automatically sets the minimum required WASI capabilities in or
 File signature:
 
 ```sh
-lucet-wasi --entrypoint main --dir /lucet:/lucet example sign LICENSE
+lucet-wasi --entrypoint main --dir .:. example sign README.md
 ```
 
 ```text
 Signature for that file: [deedf3910d5b166ca17e0e307312a422cb50efcbcc90754cf0e2d528a9159c4ad3ac973e3cd9b2c2986fb2e467a0506bc9a5ceb9c7d6d30e360fb4d1cef3c50d]
 ```
 
-This command reads the `/lucet/LICENSE` file, as well as the key pair, and computes a signature of the file's content that can be verified using the public key.
+This command reads the `README.md` file, as well as the key pair, and computes a signature of the file's content that can be verified using the public key.
 
 Signature verification:
 
 ```sh
-lucet-wasi --entrypoint main --dir /lucet:/lucet example verify LICENSE 94b8eb14373eb245c1daaacb2c24e2cb554bdd723009423aae5a8ca5fa99fa16 deedf3910d5b166ca17e0e307312a422cb50efcbcc90754cf0e2d528a9159c4ad3ac973e3cd9b2c2986fb2e467a0506bc9a5ceb9c7d6d30e360fb4d1cef3c50d
+lucet-wasi --entrypoint main --dir .:. example verify README.md 94b8eb14373eb245c1daaacb2c24e2cb554bdd723009423aae5a8ca5fa99fa16 deedf3910d5b166ca17e0e307312a422cb50efcbcc90754cf0e2d528a9159c4ad3ac973e3cd9b2c2986fb2e467a0506bc9a5ceb9c7d6d30e360fb4d1cef3c50d
```

Comment:
Since we're now using the README as the example, we should add a note here saying that you should use the output from the `sign` command here rather than this literal signature, which will fail to verify.

Reply:
Actually, I couldn't get it to verify even after using the signature produced by `sign`. Is there a trick to get it to use `keypair.bin` for the `verify` step?

Reply:
Mmm... seems to work here:

```sh
$ lucet-wasi --entrypoint main --dir .:. example keypair
Creating a new keypair...
Key pair created and saved into [keypair.bin]
Public key: [6ec6ad279c6335382fa72a6ee23dbcddf835dc438d248b4a224425e04b4abb61]

$ lucet-wasi --entrypoint main --dir .:. example sign README.md
Signature for that file: [4cbae15aeb812c642dbbd40e32c50c7bcdaf8e75519c220508407918645011152145c941ff7c451f6b66c70fdada82fa0e2c90678dff93bed58332ef0b6cf005]

$ lucet-wasi --entrypoint main --dir .:. example verify README.md 6ec6ad279c6335382fa72a6ee23dbcddf835dc438d248b4a224425e04b4abb61 4cbae15aeb812c642dbbd40e32c50c7bcdaf8e75519c220508407918645011152145c941ff7c451f6b66c70fdada82fa0e2c90678dff93bed58332ef0b6cf005
This is a valid signature for that file
```

The public key can be extracted from `keypair.bin`, but the verifier usually doesn't have that file.

Reply:
> Since we're now using the README as the example, we should add a note here saying that you should use the output from the sign command here rather than this literal signature, which will fail to verify.

Done.

Reply:
Ah, I was using the wrong public key. Got it!